src/main/java/net/mcreator/preferences/data/BlocklySection.java

```diff
@@ -51,8 +54,11 @@ public class BlocklySection extends PreferencesSection {
 		enableComments = addEntry(new BooleanEntry("enableComments", true));
 		enableCollapse = addEntry(new BooleanEntry("enableCollapse", true));
 		enableTrashcan = addEntry(new BooleanEntry("enableTrashcan", true));
+		maxTrashcanSize = addEntry(new IntegerEntry("maxTrashcanSize", 32, 1, 100));

```

Comment:
maxTrashcanContents is better name and also used by Blockly. Size first hints on visual size, but this is not the case